tools/cldr-apps/js/src/esm/cldrInfo.mjs

```diff
@@ -521,7 +521,7 @@ function updateRowVoteInfo(tr, theRow) {
     } else {
       cldrVote.appendItem(
         valdiv,
-        value,
+        item.value, // get display value not raw
```

Comment:
is appendItem consistently called with display value elsewhere?

btw I happened to notice just now, there's a line in cldrVote.mjs (not due to this PR) that has no effect:

`var span = appendItem(h3, value, "value");`

Reply:
> is appendItem consistently called with display value elsewhere?

Everywhere except i'm not sure about cldrOldVotes.mjs, but I haven't checked that API.

> btw I happened to notice just now, there's a line in cldrVote.mjs (not due to this PR) that has no effect:
> 
> `var span = appendItem(h3, value, "value");`

The `var span = ` could be removed, but it _does_ have an effect. The span will be appended to `h3` even though the return type is not captured.

Reply:
my impression is the answer is "yes" -- appendItem is consistently called with display value elsewhere

this instance was exceptional since here value is from vr.value_vote

Reply:
@btangmu ok to merge then?

Reply:
about `var span = ...` we should delete `var span` (but keep` appendItem...`) next time we touch that file